packages/o365_metrics/data_stream/teams_call_quality/fields/fields.yml

```
@@ -2,233 +2,73 @@
-              type: keyword
-              description: The header value of the user agent
-            - name: platform
-              type: keyword
-              description: The platform of the callee (e.g., Windows, macOS)
-            - name: product_family
-              type: keyword
-              description: The product family of the callee (e.g., Teams, Skype)
-        - name: cpu_cores.count
-          type: long
-          description: The number of CPU cores on the callee's device
-        - name: cpu_name
+          description: Unique identifier for the session.
+        - name: modalities 
           type: keyword
-          description: The name of the CPU on the callee's device
-        - name: cpu_processor_speed.mhz
-          type: long
-          description: The processor speed in MHz on the callee's CPU
-        - name: feedback
+          description: List of modalities present in the session. Possible values are- unknown, audio, video, videoBasedScreenSharing, data, screenSharing, unknownFutureValue.
+        - name: segments
           type: group
-          description: Feedback provided by the callee during the call
+          description: The list of segments involved in the session.
           fields:
-            - name: rating
+            - name: segment_id
               type: keyword
-              description: The rating the callee gave for the call quality
-            - name: tokens
-              type: group
-              description: Feedback tokens from the callee
-              fields:
-                - name: distorted_sound
-                  type: boolean
-                  description: Whether distorted sound was reported by the callee
-                - name: echo
-                  type: boolean
-                  description: Whether echo was reported by the callee
-                - name: interruptions
-                  type: boolean
-                  description: Whether interruptions were reported by the callee
-                - name: low_volume
-                  type: boolean
-                  description: Whether low volume was reported by the callee
-                - name: no_sound
-                  type: boolean
-                  description: Whether no sound was reported by the callee
-                - name: noisy
-                  type: boolean
-                  description: Whether background noise was reported by the callee
-                - name: other_no_sound
-                  type: boolean
-                  description: Whether other no sound issues were reported by the callee
-                - name: stopped
-                  type: boolean
-                  description: Whether the call was stopped prematurely by the callee
-        - name: identity
-          type: group
-          description: Identity of the callee
-          fields:
-            - name: user
+              description: Unique identifier for the segment              
```

nit: Can you add period for the descriptions to maintain consistency?  
```suggestion
              description: Unique identifier for the segment.              
```